cmd/options.go

```diff
@@ -218,9 +224,11 @@ func Parse() *Options {
 	viper.SetDefault("argocd-chart-url", DefaultArgocdChartURL)
 	viper.SetDefault("log-format", DefaultLogFormat)
 	viper.SetDefault("title", DefaultTitle)
+	viper.SetDefault("dry-run", DefaultDryRun)
 
 	// Basic flags
 	rootCmd.Flags().BoolP("debug", "d", false, "Activate debug mode")
+	rootCmd.Flags().BoolP("dry-run", "n", DefaultDryRun, "Show which applications would be processed without creating a cluster or generating a diff")
```

Comment:
```suggestion
	rootCmd.Flags().Bool("dry-run", DefaultDryRun, "Show which applications would be processed without creating a cluster or generating a diff")
```
I would prefer not to have a shorthand for this one :)